src/Error/ErrorTrap.php

```diff
@@ -24,9 +24,7 @@
 class ErrorTrap
 {
     use EventDispatcherTrait;
-    use InstanceConfigTrait {
-        getConfig as private _getConfig;
-    }
+    use InstanceConfigTrait;
```

Comment:
I was trying to reduce the size of the public API for the trap classes but having public read accessors are pretty low risk.